src/guiguts/root.py

```diff
@@ -73,15 +86,45 @@ def _save_config(self) -> None:
 
         Several calls to this may be queued by config changes during
         root dialog creation and resizing. Only the first will actually
-        do a save, because the flag will only be true on the first call."""
-        if self.save_config:
-            # Bug in maximized geometry leads to size being full screen size,
-            # but top-left is non-maximized top left, i.e. mid-screen.
-            # So, if maximized, don't save geometry, just save "normal/zoomed" state.
-            # Then when de-maximize happens, you have the correct size AND top-left.
-            if self.state() != "zoomed":
+        do a save, because the flag will only be true on the first call.
+
+        Will do nothing until enabled via a call to set_zoom_fullscreen."""
```

Comment:
I realize that the word `zoom`/`zoomed` was already there. I'm mostly commenting here because I think the word is being misused here. Zooming in, to me, means choosing a specific spot and doing the equivalent of looking at it via a magnifying glass. That is, whether it's an image or text, the area being zoomed in on is being enlarged.

However, if y'all think that it's an appropriate use case, I'll defer to you. I just wanted to raise awareness because we will eventually need to deal with image zoom.

Reply:
I agree that "zoom" normally means something else. In this case though, the state reported by the window is actually the word "zoomed", so it makes sense to use that within the code.